src/components/__tests__/picker-test.jsx

```diff
@@ -72,6 +72,24 @@ describe("Picker", function() {
     })
   })
 
+  describe("when a picker's clear selection button is clicked", function() {
+    let onExit      = sinon.spy()
+    let onChange    = sinon.spy()
```

Comment:
Haha, good call. Sorry to begin that convention. 

![](http://vignette1.wikia.nocookie.net/villains/images/9/9d/Spy-vs-spy.jpg/revision/latest?cb=20130504071513)